tests/pytest_tests/system_tests/test_importers/test_wandb/conftest.py

```diff
@@ -0,0 +1,336 @@
+import logging
+import os
+import random
+import string
+import subprocess
+import tempfile
+import typing
+
+import numpy as np
+import pandas as pd
+import plotly.express as px
+import plotly.io as pio
+import pytest
+import wandb
+import wandb.apis.reports as wr
+from PIL import Image
+from rdkit import Chem
+
+from ...testcontainer_setup_utils import (
+    FIXTURE_SERVICE_PORT,
+    LOCAL_BASE_PORT,
+    SERVICES_API_PORT,
+    WandbServerSettings,
+    spin_wandb_server,
+)
+
+# `local-testcontainer2` url and ports
+# Normally these env vars are only set in CI
+default_server_url2 = (
+    os.getenv("WANDB_TEST_SERVER_URL2") if os.getenv("CI") else "http://localhost"
+)
+local_base_port2 = LOCAL_BASE_PORT if os.getenv("CI") else "9180"
+service_api_port2 = SERVICES_API_PORT if os.getenv("CI") else "9183"
+fixture_service_port2 = FIXTURE_SERVICE_PORT if os.getenv("CI") else "9115"
```

Comment:
all of these only used in `pytest_configure` why make them module variables? can move it to the function.

Reply:
was meant to pattern match how `testcontainer_setup_utils.py` looks like, but maybe not necessary

Reply:
why the ports need to be different for CI?